CHANGELOG.md

```diff
@@ -1,9 +1,14 @@
 Changelog
 =========
 
+# v1.7.0
```

Comment:
Can you circle back and add dates to these?

For example, https://github.com/yarpc/yarpc-go/blob/dev/CHANGELOG.md.